esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -507,6 +509,7 @@ define(
         OrphanFile: "Archivos huérfanos",
         OrphanFile2: "Archivo huérfano",
         OrphanMessage: "Un archivo huérfano tiene solo algunas de sus partes en disco. Sin embargo, el conjunto completo de partes no esta disponible para construir un archivo logico completo. No hay referencias a estas partes de archivo en el servidor Dali.",
+        OSStats: "Estadísticas de OS",
```

Comment:
very minor, but I'm not sure if OS translate directly, perhaps "del sistema operativo"?